src/main/java/Duke.java

```diff
@@ -1,10 +1,88 @@
+import java.util.Scanner;
+
 public class Duke {
+    public static final int CAPACITY = 100;
+
     public static void main(String[] args) {
+        printWelcomeMessage();
+
+        Task[] tasks = new Task[CAPACITY];
+        int counter = 0;
```

Comment:
Would it be better if this variable was more specific? (ie taskCounter instead of counter)